File: years/2016/elixir/lib/aoc/day_03.ex
defmodule AoC.Day03 do
  @moduledoc false

  def run do
    IO.puts("day 03 part 1: #{AoC.Day03.part_1("../data/03.txt")}")
    IO.puts("day 03 part 2: #{AoC.Day03.part_2("../data/03.txt")}")
  end

  def part_1(filename) do
    filename
    |> File.stream!()
    |> Enum.count()
  end

  def part_2(filename) do
    filename
    |> File.stream!()
    |> Enum.count()
  end
end

File: years/2015/python/aoc/day_03.py
#!env python
# code: language=python insertSpaces=true tabSize=4

import io
import json

##
## day 03 part 1
##


def part1(data):
    return None


##
## day 03 part 2
##


def part2(data):
    return None


##
## main
##


def main():
    with io.open("../data/03.txt") as f:
        data = f.readlines()
        output = {"day_03": [part1(data), part2(data)]}
        print(json.dumps(output, indent=2))


##
## execution begins here
##

if __name__ == "__main__":
    main()
